### PR TITLE
Introduce first example of handling data nicely

### PR DESCRIPTION
**Pull request by rotwurstesser**
# Checklist (Only for frontend changes)
- [ ] Changes tested in IE11 (not needed)
- [ ] Responsive (not needed)
- [x] Documented "How to use it"
- [x] Documented "How to install"
- [ ] Featured in Styleguide (not needed)

# Summary of this PR

### CODE REVIEW

File: docs/frontend/ConnectingApis.md [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+    model.planet = data.planet
+  }else {
+    model.planet = null
+  }
+  return model;
+}
+```
+In this case we can always go sure that we have a name to display, even if it is "No Name defined". We also know for sure that if our planet has no name we will have the planet set to `null` which we can use for easier checks in the template like this:
+```vue
+<h2 v-if='alien.planet'>{{alien.planet.planetName}}</h2>
```
Comment (JonasNiestroj) [inline docs/frontend/ConnectingApis.md]: Maybe we should also add the check for `alien != null`.
The model returns null if data is not defined so there is the possibility for alien to be null.

File: frontend/src/examples/APIUsage.vue [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+  data() {
+    return {
+      card: null,
+    };
+  },
+  async mounted() {
+    try {
+      let cardData = await getMagicCard();
+      this.card = {};
+      this.card.name = cardData.name;
```
Comment (JonasNiestroj) [inline frontend/src/examples/APIUsage.vue]: Is there a reason for just not writing `this.card = cardData`? If there are more properties its a lot easier.

File: frontend/src/services/mtg-card.service.js [hunk 3]
```diff
@@ -0,0 +1,12 @@
+import { mtgApi } from './api';
+import { MtgCard } from '../models/mtg-card.model';
+
+const getMagicCard = async function() {
+  // you can add things like bearertokens or payloads here
+  let config = { crossdomain: true };
+  const response = await mtgApi.get('cards/random', config);
+  const result = MtgCard(response.data);
```
Comment (JonasNiestroj) [inline frontend/src/services/mtg-card.service.js]: Just Javascript Things: In JS `MtgCard` and `new MtgCard` is usable. (I think we should use `new MtgCard` so its easier to understand that it is used as a model)

Comment (rotwurstesser) [reply frontend/src/services/mtg-card.service.js]: true that!

File: frontend/src/examples/APIUsage.vue (hunk 2 above)
Comment (rotwurstesser) [reply frontend/src/examples/APIUsage.vue]: nope, no reason, added that!

File: docs/frontend/ConnectingApis.md (hunk 1 above)
Comment (rotwurstesser) [reply docs/frontend/ConnectingApis.md]: yep, added x)

**Pull request merged**